content/rc/subscriptions/view-fixed-subscription.md

```diff
@@ -41,26 +41,26 @@ The following sections provide more details.
 
 ## Upgrade subscription plan
 
-Use the **Upgrade plan** button to update your Fixed subscription tier, your high availability settings, or your payment method.
+Use the **Upgrade plan** button to update your Fixed subscription plan, your high availability settings, or your payment method.
```

Comment:
@cmilesb I think it is worthwhile to point out here that upgrading the database does not impact its availability during the update (Fixed to Fixed)

Reply:
```suggestion
Use the **Upgrade plan** button to update your Fixed subscription plan, your high availability settings, or your payment method. Upgrading your database between Fixed plans does not impact database availability during the update.
```